docker-compose.yml

```diff
@@ -71,6 +71,7 @@ balrogui:
     - .cache:/cache
   environment:
     - WEB_PORT=8080
+    - LOG_LEVEL=WARNING
```

Comment:
I have implemented the changes. The log level for ui and db containers have been removed.